src/webots/vrml/WbNode.cpp

```diff
@@ -827,7 +827,7 @@ WbField *WbNode::findSubField(int index, WbNode *&parentNode) const {
   foreach (WbNode *const node, list) {
     foreach (WbField *const field, node->mFields) {
       if (count == index) {
-        parentNode = node;
+        parent = node;
```

Comment:
```suggestion
        parentNode = node;
```
This change is wrong.

Reply:
Well spotted! But in that case, I would prefer to rename the parameter name as the `parentNode` parameter name shadows the `parentNode()` method.